lib/test/CMakeLists.txt

```diff
@@ -644,6 +644,7 @@ ot_check_test (GeneralLinearModelAlgorithm_std)
 ot_check_test (TensorApproximationAlgorithm_std)
 ot_check_test (LinearModelAlgorithm_std)
 ot_check_test (LinearModelAnalysis_std)
+ot_check_test (KrigingAlgorithm_isotropic_std)
```

Comment:
`ot_check_test (KrigingAlgorithm_isotropic_std IGNOREOUT)`
And  drop the `expout` file